site/docs/reference/Connectors/capture-connectors/PostgreSQL.md

```diff
@@ -109,3 +117,64 @@ The PostgreSQL source connector [supports SSH tunneling](../../../concepts/conne
 to allow Flow to connect to databases ports in secure networks.
 
 To set up and configure your SSH server, see the [guide](../../../../guides/connect-network/).
+
+## PostgreSQL on Amazon RDS
+
+Amazon Relational Database Service (RDS) is a managed web service providing cloud-based instances
+of popular relational databases, including PostgreSQL.
+
+### Setup
+
+You can use this connector for PostgreSQL instances on RDS, but the setup requirements are different.
+
+1. Create a new RDS PostgreSQL instance for testing. You can
```

Comment:
I would remove the first step here. In the typical case the user already has a DB with a bunch of data in it that they need captured into Flow, and starting with "here's how you create a database" is just going to cause confusion.

I just included this part in the Slack thread because that's where my notes began, and because some of those settings were non-obvious when I started.

Reply:
Got it, my bad- i was reading and assuming we'd want them to set up a test first before hooking up the production DB but that wouldn't really make sense. 
So for pre-existing DBs, are there any of these settings that are important to change? Specifically thinking of Public Access?

Reply:
No, I don't think any of these settings need to be mentioned. The "Public Access" thing was part of making it directly reachable from an external IP. Since we're going to steer users towards an SSH proxy for access that shouldn't be necessary.